pkg/operator/hive/hive.go

```diff
@@ -227,7 +226,7 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 	for _, isName := range removeImageSets {
 		clusterImageSet := &hivev1.ClusterImageSet{}
 		err := r.Get(context.Background(), types.NamespacedName{Name: isName}, clusterImageSet)
-		if err != nil && !errors.IsNotFound(err) {
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Although not really related to your changes, the `err != nil` is unnecessary.

Reply:
This boils down to a is err.(type) which makes me nervous but how about we just delete the whole block as it's ancient cleanup. :)

Reply:
Even better!

Although I would like to know more about what makes you nervous about removing the `err != nil`.

Reply:
a nil err and err.(type) is not something i would ever see and assume was ok. It appears it's fine but it just looks like something that could cause a panic and crash. It reads better IMO with the != nil guard.

Reply:
OK. It is guaranteed from the language to not panic. It is perfectly valid to do nil.(type).